Adadie/Fresh
# #19-change-display-styles

#### What does this PR do?
This PR will update styles of all the pages.
#### Description of Task to be completed?
- Change padding and margin of all web pages.
- Increase comments input box size.
- Increase media query screen size.
#### How should this be manually tested?
- Clone this repository in your local repo [link](https://github.com/Junior-Adadie/Fresh.git).
- Open index.html with your browser.
- Check if the comments input box height is higher than the message box.
- Make sure your browser is updated.
- Check if no excess unnecessary padding or margin is on any page.
#### Any background context you want to provide?
The top padding of every page was 200px and the media query screen size was 768Px.
#### What are the relevant Trello stories?
[Trello Card](https://trello.com/c/30WoVlEU/19-change-display-styles)
#### Screenshots (if appropriate)
![Capture2](https://user-images.githubusercontent.com/68109866/88489034-64b94c80-cf9a-11ea-8472-c9a73be6a16a.PNG)
![Capture](https://user-images.githubusercontent.com/68109866/88489040-71d63b80-cf9a-11ea-9171-cc2941a913a3.PNG)
